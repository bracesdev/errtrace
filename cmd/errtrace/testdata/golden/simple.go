//go:build ignore

package foo

import "strconv"

func Unwrapped(s string) (int, error) {
	i, err := strconv.Atoi(s)
	if err != nil {
		return 0, err
	}
	return i + 42, nil
}

// TODO: multiple return values without variables
