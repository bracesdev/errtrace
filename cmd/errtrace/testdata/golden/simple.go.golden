//go:build ignore

package foo

import "strconv"; import "braces.dev/errtrace"

func Unwrapped(s string) (int, error) {
	i, err := strconv.Atoi(s)
	if err != nil {
		return 0, errtrace.Wrap(err)
	}
	return i + 42, nil
}

// TODO: multiple return values without variables
